ddtrace/__init__.py

```diff
@@ -71,3 +76,17 @@ def __getattr__(name):
         return globals()[name]
 
     raise AttributeError("%s has no attribute %s", __name__, name)
+
+
+def check_supported_python_version():
+    if PYTHON_VERSION_INFO < (3, 8):
+        debtcollector.deprecate(
+            (
+                "Support for ddtrace with Python version %d.%d is deprecated and will be removed in a future release."
```

Comment:
Should we be specifying which major version? Maybe specify 2.0.0 for <3.7 and 3.0.0 for <3.8?

Reply:
I believe we "dropped" Python 3.7 support in 2.0.0, but I don't remember much, maybe @majorgreys remembers?

Reply:
Yeah this is just a nit, feel free to disregard if you think it's more confusing than helpful. "Future version" just seemed a bit vague so thought I'd make the suggestion 😅